spec/inputs/snmp_spec.rb

```diff
@@ -15,6 +15,7 @@
     before do
       expect(LogStash::SnmpClient).to receive(:new).and_return(mock_client)
       expect(mock_client).to receive(:get).and_return({})
+       expect(mock_client).to receive(:close)
```

Comment:
Nit: indentation mismatch

Reply:
I removed that, I will file an issue against devutil because the `TestPipeline` does not close the plugins.